mlir/include/mlir/Dialect/MIOpen/gridwise_gemm_params.h

```diff
@@ -563,17 +563,24 @@ struct InitParamsNonXDL : InitParams {
   int64_t blockSize;
 };
 
+// block gemm tuning params that sepcific the layout of thread-wise gemm in a
+// workgroup
+struct DerivedBlockGemmParams {
+	int64_t gemmMLevel0Cluster;
+	int64_t gemmNLevel0Cluster;
+	int64_t gemmMLevel1Cluster;
+	int64_t gemmNLevel1Cluster;
+};
+
 class PopulateParams : public PopulateParamsBase {
 private:
   // clang-format off
   llvm::SmallVector<InitParamsNonXDL, 4> initParameters = {
       // M/block N/block K/block M/thread N/thread blockSize
-      {128, 128, 8, 4, 4, 256},
-      {128, 64, 8, 4, 4, 128},
-      {64, 128, 4, 4, 4, 128},
-      {64, 64, 16, 4, 4, 64},
-      {32, 64, 16, 2, 4, 64},
-      {32, 32, 4, 2, 2, 64},
+	  {128, 128, 8, 4, 4, 256}, {128, 64, 8, 4, 4, 128},
```

Comment:
Strange that the code formatting changed between // clang-form off/on. Could you revert the original single line format?